### ISSUE TITLE
Refactor `getSync`

### ISSUE THREAD

**Issue by kachnitel** [kachnitel/RideTime#137]
Return entity populated with ID only, call populate in background

```
  getSync (id: Number) {
    let entity = this._findInCollection(id)
    if (!entity) {
      entity = new this.EntityClass()
      entity.updateId(id)
      entity.populateFromApi(id)
    }
    return entity
  }
```

- Catch error and fill with dummy error information, log error
  - In an async wrapper for `populateFromApi`

**Comment by kachnitel** [comment]
a56fe07
- Renamed `getSync` to `get`
- No longer throw an error and lazy load the entity instead

**Comment by kachnitel** [comment]
Implemented in a56fe07
